src/Razor/src/Microsoft.CodeAnalysis.Razor.Workspaces/Completion/DirectiveAttributeCompletionItemProviderBase.cs

```diff
@@ -11,37 +11,12 @@ namespace Microsoft.CodeAnalysis.Razor.Completion
 {
     internal abstract class DirectiveAttributeCompletionItemProviderBase : RazorCompletionItemProvider
     {
-        // Internal for testing
-        internal static bool IntersectsWithAttributeNameOrPrefix(SourceSpan location, TextSpan? prefixLocation, TextSpan attributeNameLocation)
```

Comment:
Moved to DirectiveAttributeCompletionTransition as that's the only consumer of this function.